pytest/tests/runtime/fuzz.py

```diff
@@ -6,7 +6,9 @@
 def run_fuzz():
     env = os.environ.copy()
     env["RUSTC_BOOTSTRAP"] = "1"
-    fuzzing = subprocess.Popen(('cargo', 'fuzz', 'run', 'runtime-fuzzer',
+    cpus = len(os.sched_getaffinity(0))
+    fuzzing = subprocess.Popen(('cargo', 'fuzz', 'run', '--jobs', cpus,
```

Comment:
Are you sure that passing cpus as integer here would work? It fails on gcp machine